examples/spring-boot-on-kubernetes-without-annotations/src/main/resources/application.properties

```diff
@@ -8,3 +8,4 @@ dekorate.kubernetes.ports[0].containerPort=8080
 dekorate.kubernetes.serviceType=LoadBalancer
 dekorate.kubernetes.configMapVolumes[0].volumeName=bar-volume
 dekorate.kubernetes.configMapVolumes[0].configMapName=foo-map
+dekorate.docker.group=dekorate.io
```

Comment:
Hi Jack! Thanks for taking the time to contribute!
Can you please provide some more details on how this makes image manipulation cleaner?

Reply:
Hi, thanks for getting back! I upgrade from 0.7 to the 0.10.11 and the image property in the generated yaml file stopped using the **dekorate.kubernetes.group** property as a prefix. I think this is right, as it wasn't ideal behaviour as stated in https://github.com/dekorateio/dekorate/issues/409

I was using amazon ecr so wanted a prefix of [ACCOUNT_NUMBER].dkr.ecr.us-east-1.amazonaws.com

It wasn't clear how to achieve this, but eventually I discovered **dekorate.docker.group** (and version etc)

It would have saved me (and others?) a lot of time if this example (spring-boot-on-kubernetes-without-annotations) had made use of **dekorate.docker.group**

At the moment, the readme.md of this example states that the generated file includes  
`image: "hello-world/spring-boot-on-kubernetes-without-annotations:0.8-SNAPSHOT"`

but it doesn't, it actually (for me jbeaken) generates **jbeaken/spring-boot-on-kubernetes-without-annotations:0.10-SNAPSHOT** 

So I created this pull request to hopefully make it clearer. If you want to proceed, I can add tests etc.

thanks!

Reply:
We definitely, need to take action and make it clear for everyone.

I'd prefer if we had a dedicated example for that, maybe something like `spring-boot-with-custom-docker-image-name` or something like that and an entry in the documentation. Is this something you would like to add?

Reply:
I should be able to do, as I don't know dekorate too well, though we will be using it in our next project. 

It would be helpful if i could use a 'base' example get me started, for example the `spring-boot-on-kubernetes-without-annotations `

Is that something you'd be happy with?

Reply:
That sound good, just scrap all other properties from application.properties and just keep `dekorate.docker.group`.

Reply:
Let's hold that thought for a while.

I am considering of completly wiping `group` our of `kubernetes`, `openshift` and `knative` configuration and just keep it around for image config like `docker` and `jib`. This will completlely demistify this.

In the place of `group` we can use `part-of` which is also the offcial recommendation of kubernetes.

Reply:
sure thing, whatever you think best

Reply:
The last commit of #459 sums up what I have in mind. Feedback is more than welcome.